custom/static/css/bootstrap-responsive.css

```diff
@@ -997,7 +997,7 @@
   }
   .navbar-inverse .nav-collapse .nav > li > a,
   .navbar-inverse .nav-collapse .dropdown-menu a {
-    color: #999999;
+    color: #444444;
```

Comment:
Please revert this change. This file isn't used by the new external web app; it's part of legacy code.